### PR TITLE
Bring back menubar without formatting on plain text mode

### PR DESCRIPTION
**Pull request by juliushaertl**
Fixes https://github.com/nextcloud/text/issues/2625

- Ensure that the session list on the right of the menubar is always visible
  - For plain text files
  - For read only shares of markdown and plain text files

### CODE REVIEW

**Comment by juliushaertl** [comment]
Rebased and ready for review :)

**Review by max-nextcloud — DISMISSED** [review]
All in all looks good. I think we do not need to render the menu bubble unless we are editing a richtext document.
With the change to `renderMenus` it would now be rendered for text. Since it even has it's own dynamic import might be worthwhile having two computed conditions `renderMenubar` and `renderMenububble` or so.

**Comment by juliushaertl** [comment]
Rebased and ready for review again :)

File: src/components/Editor.vue [hunk 1]
```diff
@@ -262,12 +269,16 @@ export default {
 		displayed() {
 			return this.currentSession && this.active
 		},
-		renderMenus() {
+		renderRichEditorMenus() {
 			return this.contentLoaded
 				&& this.isRichEditor
 				&& !this.syncError
 				&& !this.readOnly
 		},
```
Comment (max-nextcloud) [inline src/components/Editor.vue:277]: Are we using this anywhere? Or are you just keeping it in case we need it later?
```suggestion
```

**Review by max-nextcloud — APPROVED** [review]
Looks good. Just one function that i think is not needed anymore. But that should not block a merge.

**Comment by max-nextcloud** [comment]
cypress was failing earlier - curious to see if this still happens.

**Comment by max-nextcloud** [comment]
Not sure if a backport to 23 is needed. We need the one to stable 24 for sure as a customer complained about this (in ios but i think it's this issue.)

File: src/components/Editor.vue (hunk 1 above)
Comment (max-nextcloud) [reply src/components/Editor.vue:277]: I guess we still need this in earlier versions that still have the menu bubble. So good to keep it for now for the backports and clean it up separately.

**Comment by max-nextcloud** [comment]
Looks like for the keyboard shortcut tests retries run into file version conflicts

**Comment by max-nextcloud** [comment]
/backport 665764231b461f392b03da93fa949133975b5be7 to stable24

**Pull request merged**